app/templates/resources/add.html

```diff
@@ -0,0 +1,127 @@
+            <h2 class="ui dividing header">Add a resource</h2>
+
+            {% set flashes = {
+                'error':   get_flashed_messages(category_filter=['form-error']),
+                'warning': get_flashed_messages(category_filter=['form-check-email']),
+                'info':    get_flashed_messages(category_filter=['form-info']),
+                'success': get_flashed_messages(category_filter=['form-success'])
+            } %}
+
+            {{ f.begin_form(form, flashes) }}
+
+                <div class="two fields">
+                    {{ f.render_form_field(form.name) }}
+                    {{ f.render_form_field(form.website) }}
+                </div>
+
+                {{ f.render_form_field(form.description) }}
+
+                {{ f.render_form_field(form.address_autocomplete) }}
+
+                <div class="two fields">
+                    {{ f.render_form_field(form.street_number) }}
+                    {{ f.render_form_field(form.route) }}
+                </div>
+
+                {{ f.render_form_field(form.locality) }}
+
+                <div class="two fields">
+                    {{ f.render_form_field(form.administrative_area_level_1) }}
+                    {{ f.render_form_field(form.postal_code) }}
+                </div>
+
+                {{ f.render_form_field(form.submit) }}
+
+                {{ f.form_message(flashes['error'], header='Something went wrong.', class='error') }}
+                {{ f.form_message(flashes['warning'], header='Check your email.', class='warning') }}
+                {{ f.form_message(flashes['info'], header='Information', class='info') }}
+                {{ f.form_message(flashes['success'], header='Success!', class='success') }}
+
+            {{ f.end_form(form) }}
+        </div>
+    </div>
+
+    <script>
+        // Set onFocus attribute for autocomplete field and disabled attribute for address fields.
+        $( document ).ready(function() {
+            $("#address_autocomplete").focus(geolocate())
+
+            for (var component in componentForm) {
+                document.getElementById(component).disabled = true;
+            }
+        })
+
+
+        // https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform
+        // This example displays an address form, using the autocomplete feature
+        // of the Google Places API to help users fill in the information.
+
+        var placeSearch, autocomplete;
+        var componentForm = {
+            street_number: 'short_name',
+            route: 'long_name',
+            locality: 'short_name',
+            administrative_area_level_1: 'short_name',
+            postal_code: 'short_name'
+        };
+
+        function initAutocomplete() {
+            // Create the autocomplete object, restricting the search to geographical
+            // location types.
+            autocomplete = new google.maps.places.Autocomplete(
```

Comment:
I'm gonna push you on the jquery because I'm in luv wit the jquery try this `$('#address_autocomplete')[0]`